lib/assessment.rb

```diff
@@ -43,6 +44,11 @@ def transform(words)
   # replacing the trailing newline character with a
   # space and return the result.
   def read_file(filename)
-    # Your code goes here
+    File.open(ARGV[0]) do |file|
+      lwc = file.each_with_object(Hash.new(0)) do |line, memo|
```

Comment:
Also, you could have chained this: 
`File.open(filename).each_with_object(Hash.new(0))`